ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/remote/AbstractConnectorExecutor.java

```diff
@@ -0,0 +1,28 @@
+package org.opensearch.ml.engine.algorithms.remote;
+
+import lombok.Getter;
+import lombok.Setter;
+
+public abstract class AbstractConnectorExecutor implements RemoteConnectorExecutor{
```

Comment:
Why not just put @Setter and @Getter on top of this class so it applies to all field.